packages/azure-kusto-data/package.json

```diff
@@ -29,25 +44,37 @@
         "directory": "azure-kusto-data"
     },
     "files": [
-        "source/**/*.js",
-        "source/**/*.d.ts",
-        "index.js",
-        "index.d.ts",
+        "dist/",
+        "dist-esm/src/",
+        "dist-esm/package.json",
         "tsconfig.tsbuildinfo",
         "example.js"
     ],
     "author": "",
     "license": "ISC",
     "homepage": "https://github.com/Azure/azure-kusto-node/blob/master/azure-kusto-data/README.md",
+    "devDependencies": {
+        "@types/mocha": "^9.1.1",
+        "@types/webpack-dev-server": "^4.7.2",
+        "exports-loader": "^4.0.0",
+        "ts-loader": "^9.3.1",
+        "webpack-cli": "^4.10.0",
+        "rimraf": "^3.0.2"
+    },
     "dependencies": {
         "@azure/identity": "^2.1.0",
-        "@azure/msal-node": "^1.12.1",
+        "@azure/msal-common": "^7.3.0",
         "@types/node": "^18.6.4",
         "@types/uuid": "^8.3.4",
+        "assert": "^2.0.0",
```

Comment:
I don't recommend using node's assert module, much less this browser package equivalent - chai provides compatible and nicer assertions -- but this should be a dev-dep too right?

Reply:
i think ill leave the assert for now and move this to dev dependencies